App.tsx

```diff
@@ -1,21 +1,28 @@
+
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
+import Signup from './components/auth/signup.component'
+import TempHome from './components/general/temphome.component'
+import {BrowserRouter as Router, Route} from 'react-router-dom'
 import { StyleSheet, Text, View } from 'react-native';
+import LogIn from './components/auth/login.component';
 
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+  return ( 
+    <Router> 
+      <View style={styles.appcontainer}>
+        <Route exact path='/' component={TempHome}/>
+        <Route path='/signup' component={Signup}/>
```

Comment:
Nit: can we make this SIgnUp just to have the same convention as the other components